### PR TITLE
Refresh the vmware vcsa restapi doc

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

EDIT by @omercier 
REFS: CTOR-169

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 1]
```diff
@@ -2,43 +2,316 @@
...
+| Métrique         | Unité |
+|:-----------------|:------|
+| *service*#status | N/A   |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:47]: sample text

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 2]
```diff
@@ -2,43 +2,316 @@
...
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Health" label="Health">
+
+| Macro          | Description                                                                                                                                                | Valeur par défaut    | Obligatoire |
+|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------|:-----------:|
+| FILTERSERVICE  | Filter service (can be a regexp)                                                                                                                           |                      |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:166]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 3]
```diff
@@ -2,43 +2,316 @@
...
+	--filter-service='' \
+	--unknown-status='' \
+	--warning-status='' \
+	--critical-status='%{health} !~ /green/' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:203]: is this ok?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 4]
```diff
@@ -2,43 +2,316 @@
...
+| --filter-perfdata-adv                      | Filter perfdata based on a "if" condition using the following variables: label, value, unit, warning, critical, min, max. Variables must be written either %{variable} or %(variable). Eg: adding --filter-perfdata-adv='not (%(value) == 0 and %(max) eq "")' will remove all metrics whose value equals 0 and that don't have a maximum value.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:257]:
```suggestion
| --filter-perfdata-adv                      | Filter perfdata based on an "if" condition using the following variables: label, value, unit, warning, critical, min, max. Variables must be written either %{variable} or %(variable). Eg: adding --filter-perfdata-adv='not (%(value) == 0 and %(max) eq "")' will remove all metrics whose value equals 0 and that don't have a maximum value.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 5]
```diff
@@ -2,43 +2,316 @@
...
+| --hostname                                 | Set hostname or IP of vsca.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:276]:
```suggestion
| --hostname                                 | Specify the hostname or IP address of the VSCA server you want to monitor.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 6]
```diff
@@ -2,43 +2,316 @@
...
+| --redis-attribute                          | Set Redis Options (--redis-attribute="cnx\_timeout=5").                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:291]:
```suggestion
| --redis-attribute                          | Set Redis options (--redis-attribute="cnx\_timeout=5").                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 7]
```diff
@@ -2,43 +2,316 @@
...
+| --filter-service                           | Filter service (can be a regexp).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:301]: ?

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 8]
```diff
@@ -2,43 +2,315 @@
...
+|:-----------------|:------|
+| *service*#status | N/A   |
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:46]: sample text

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 9]
```diff
@@ -2,43 +2,315 @@
...
+
+1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
+2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).
+
+<Tabs groupId="sync">
+<TabItem value="Health" label="Health">
+
+| Macro          | Description                                                                                                                                                | Default value        | Mandatory   |
+|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------|:-----------:|
+| FILTERSERVICE  | Filter service (can be a regexp)                                                                                                                           |                      |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:167]: ?

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 10]
```diff
@@ -2,43 +2,315 @@
...
+| --filter-service                           | Filter service (can be a regexp).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:300]: ?

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 11]
```diff
@@ -2,43 +2,315 @@
...
+| --filter-perfdata-adv                      | Filter perfdata based on a "if" condition using the following variables: label, value, unit, warning, critical, min, max. Variables must be written either %{variable} or %(variable). Eg: adding --filter-perfdata-adv='not (%(value) == 0 and %(max) eq "")' will remove all metrics whose value equals 0 and that don't have a maximum value.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:256]:
```suggestion
| --filter-perfdata-adv                      | Filter perfdata based on an "if" condition using the following variables: label, value, unit, warning, critical, min, max. Variables must be written either %{variable} or %(variable). Eg: adding --filter-perfdata-adv='not (%(value) == 0 and %(max) eq "")' will remove all metrics whose value equals 0 and that don't have a maximum value.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 12]
```diff
@@ -2,43 +2,315 @@
...
+| --hostname                                 | Set hostname or IP of vsca.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:275]:
```suggestion
| --hostname                                 | Specify the hostname or IP address of the VSCA server you want to monitor.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 13]
```diff
@@ -2,43 +2,315 @@
...
+| --redis-attribute                          | Set Redis Options (--redis-attribute="cnx\_timeout=5").                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:290]:
```suggestion
| --redis-attribute                          | Set Redis options (--redis-attribute="cnx\_timeout=5").                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 14]
```diff
@@ -2,43 +2,315 @@
...
+	--unknown-status='' \
+	--warning-status='' \
+	--critical-status='%{health} !~ /green/' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All services are ok | '*service*#status'=;;;;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:203]: is this ok?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 15]
```diff
@@ -2,43 +2,310 @@
...
 
-| Macro           | Description              | Default value |
-| :-------------- | :----------------------- | :------------ |
-| VCSAAPIPORT     | Port used by the API     | 443           |
-| VCSAAPIPROTO    | Protocol used by the API | https         |
-| VCSAAPIUSERNAME | Username to access API   |               |
-| VCSAAPIPASSWORD | Password to access API   |               |
+| Macro          | Description                                                                                                                                                | Valeur par défaut    | Obligatoire |
+|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------|:-----------:|
+| FILTERSERVICE  | Filter service (can be a regexp)                                                                                                                           |                      |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:160]:
```suggestion
| FILTERSERVICE  | Define which services should be monitored based on the services' names. This option will be treated as a regular expression.                                                                                                                        |                      |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 16]
```diff
@@ -2,43 +2,310 @@
...
+| --filter-service                           | Filter service (can be a regexp).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:295]:
```suggestion
| --filter-service                           | Define which services should be monitored based on the services' names. This option will be treated as a regular expression.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 17]
```diff
@@ -2,43 +2,309 @@
...
+| --filter-service                           | Filter service (can be a regexp).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:294]:
```suggestion
| --filter-service                           | Define which services should be monitored based on the services' names. This option will be treated as a regular expression.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 18]
```diff
@@ -2,43 +2,309 @@
...
 
-| Macro           | Description              | Default value |
-| :-------------- | :----------------------- | :------------ |
-| VCSAAPIPORT     | Port used by the API     | 443           |
-| VCSAAPIPROTO    | Protocol used by the API | https         |
-| VCSAAPIUSERNAME | Username to access API   |               |
-| VCSAAPIPASSWORD | Password to access API   |               |
+| Macro          | Description                                                                                                                                                | Default value        | Mandatory   |
+|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------|:-----------:|
+| FILTERSERVICE  | Filter service (can be a regexp)                                                                                                                           |                      |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:161]:
```suggestion
| FILTERSERVICE  | Define which services should be monitored based on the services' names. This option will be treated as a regular expression.                                                                                                                        |                      |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 19]
```diff
@@ -2,43 +2,310 @@
 id: applications-vmware-vcsa-restapi
 title: VMware VCSA RestAPI
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Contenu du pack
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:8]:
```suggestion
vCenter Server Appliance (VCSA) est une machine virtuelle Linux préconfigurée et optimisée pour l'exécution de VMware vCenter Server et des services associés sur Linux.

## Contenu du pack
```

File: pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md [hunk 20]
```diff
@@ -2,43 +2,309 @@
 id: applications-vmware-vcsa-restapi
 title: VMware VCSA RestAPI
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Pack assets
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-vmware-vcsa-restapi.md:8]:
```suggestion
The vCenter Server Appliance (VCSA) is a preconfigured Linux virtual machine, which is optimized for running VMware vCenter Server and the associated services on Linux.

## Pack assets
```

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**